package.json

```diff
@@ -41,8 +41,11 @@
   },
   "devDependencies": {
     "babel-cli": "6.11.4",
-    "babel-preset-es2015": "6.6.0",
+    "babel-core": "^6.18.2",
```

Comment:
Please, don't use `^` version to ensure environments are stucked to specific versions. Can you remove them from the package.json file?